discourse-doctor.rb

```diff
@@ -55,7 +63,7 @@ def check_smtp_config
 def grep_logs
   info("Search logs for errors...")
 
-  system("grep -E -w \"error|warning\" /var/www/discourse/log/production.log | sort | uniq -c | sort -r")
+  system("grep -E -w \"error|warning\" #{DISCOURSE_PATH.join("log/production.log")} | sort | uniq -c | sort -r")
```

Comment:
Why don't you like `Pathname`? Because its not in core? I find explicit types better than using strings for everything.